### PR TITLE
docs:update configuration metadata.

### PR DESCRIPTION
**Pull request by SkyeBeFreeman**
## PR Type

Documentation content changes.

## Checklist

- [x] Add copyright holder at the beginning of .class file if it is new.
- [x] Add information of this PR to CHANGELOG.md in root of project.
- [x] All junit tests passing.
- [ ] Coverage from `Codecov Report` should not decrease.

## Checklist (Optional)

- [x] Will Pull Request to branch of 2020.0 and 2021.0.
- [x] Add documentation in javadoc in code or comment below the PR if necessary.
- [x] Add your name as @author to the beginning of .class file.

### CODE REVIEW

**Review by misselvexu — APPROVED** [review]
LGTM

File: spring-cloud-starter-tencent-polaris-discovery/src/main/java/com/tencent/cloud/polaris/PolarisDiscoveryProperties.java [hunk 1]
```diff
@@ -235,7 +235,7 @@ public String toString() {
 				", enabled=" + enabled +
 				", registerEnabled=" + registerEnabled +
 				", heartbeatEnabled=" + heartbeatEnabled +
-				", heartBeatInterval=" + heartBeatInterval +
+				", heartBeatInterval=" + heartbeatInterval +
```
Comment (misselvexu) [inline spring-cloud-starter-tencent-polaris-discovery/src/main/java/com/tencent/cloud/polaris/PolarisDiscoveryProperties.java:238]: Rename to `heartbeatInterval` .

**Review by misselvexu — CHANGES_REQUESTED** [review]

File: spring-cloud-starter-tencent-polaris-discovery/src/main/java/com/tencent/cloud/polaris/PolarisDiscoveryProperties.java (hunk 1 above)
Comment (SkyeBeFreeman) [reply spring-cloud-starter-tencent-polaris-discovery/src/main/java/com/tencent/cloud/polaris/PolarisDiscoveryProperties.java:238]: OK.

Comment (SkyeBeFreeman) [reply spring-cloud-starter-tencent-polaris-discovery/src/main/java/com/tencent/cloud/polaris/PolarisDiscoveryProperties.java:238]: > Rename to `heartbeatInterval` .

It is already `heartbeatInterval` .

**Review by misselvexu — APPROVED** [review]

**Pull request merged**